app/models/course.rb

```diff
@@ -144,11 +144,11 @@ def new_draft_attributes
           class_name: "CourseEnrichment"
 
   scope :within, ->(range, origin:) do
-    joins(:sites).merge(Site.within(range, origin: origin))
+    joins(site_statuses: :site).merge(SiteStatus.where(status: :running)).merge(Site.within(range, origin: origin))
   end
 
   scope :by_distance, ->(origin:) do
-    joins(:sites).merge(Site.by_distance(origin: origin))
+    joins(site_statuses: :site).merge(SiteStatus.where(status: :running)).merge(Site.by_distance(origin: origin))
   end
```

Comment:
Is this code rot whereby its leftovers and never used?

Reply:
Do you mean we could drop the `Course#sites` association? It's hard to tell where it's used!